tailor_client/src/lib.rs

```diff
@@ -156,3 +155,21 @@ impl<'a> TailorConnection<'a> {
         Ok(self.profiles.reload().await?)
     }
 }
+
+impl<'a> TailorConnection<'a> {
+    /// Temporarily override the performance profile. Please note that this will not survive a
+    /// restart as the performance profile is handled by the overall profile configuration.
+    pub async fn set_profile(&self, name: &str, value: &str) -> ClientResult<()> {
+        Ok(self.performance.set_profile(name, value).await?)
+    }
+
+    /// Read the current performance profile.
+    pub async fn get_profile(&self, name: &str) -> ClientResult<String> {
+        Ok(self.performance.get_profile(name).await?)
+    }
+
+    /// Read the list of supported performance profiles.
+    pub async fn list_profiles(&self) -> ClientResult<Vec<String>> {
+        Ok(self.performance.list_profiles().await?)
+    }
+}
```

Comment:
Ok, so one last thing :sweat_smile: 

Ideally, to match the existing convention, those methods should be called `set_performance_profile`, `get_performance_profile` and `list_performance_profile`.